pex/pip.py

```diff
@@ -17,19 +17,138 @@
 from pex.network_configuration import NetworkConfiguration
 from pex.third_party import isolated
 from pex.tracer import TRACER
-from pex.typing import TYPE_CHECKING, cast
+from pex.typing import TYPE_CHECKING
 from pex.variables import ENV
 
 if TYPE_CHECKING:
-    from typing import Iterator, List, Optional, Tuple, Mapping
+    from typing import Iterable, Iterator, List, Mapping, Optional, Tuple
+
+
+class PackageIndexConfiguration(object):
+    @staticmethod
+    def _calculate_args(
```

Comment:
This method was moved from https://github.com/pantsbuild/pex/pull/1063/files#diff-77bac56887c1a03806e247fe60c2024cL129 but otherwise unchanged save for the top comment.